xbmc/cores/dvdplayer/DVDCodecs/Video/AMLCodec.cpp

```diff
@@ -1427,6 +1371,11 @@ CAMLCodec::CAMLCodec() : CThread("CAMLCodec")
   am_private = new am_private_t;
   memset(am_private, 0, sizeof(am_private_t));
   m_dll = new DllLibAmCodec;
+  if(!m_dll->Load())
+  {
+    CLog::Log(LOGDEBUG, "CAMLCodec::CAMLCodec libamplayer.so not found, trying libamcodec.so instead");
+    m_dll->SetFile("libamcodec.so");
+  }
   m_dll->Load();
```

Comment:
While this should work anyway (if the first `Load()` call was successful, the second one should return immediately), it looks kinda strange, and i'll move it into the `if` conditional.